src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
+      <div class="moyo_header_logo">
+        <a
+          href="#"
+          class="menu_link"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+      <nav>
+        <ul class="moyo_menu">
```

Comment:
The class name 'moyo_menu' could be more semantic. Consider using 'nav__list' or another meaningful name that describes the role of the element.